sql/14-add-marketplace-table.sql

```diff
@@ -0,0 +1,22 @@
+\connect blobber_meta;
```

Comment:
You might consider using auto migration with gorm instead of manually generating tables

Reply:
Will definitely take a look to find which is beetter